demisto_sdk/commands/validate/validators/ST_validators/ST111_no_exclusions_schema.py

```diff
@@ -0,0 +1,53 @@
+from typing import Iterable
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = Integration
+
+ALLOWED_SECTIONS = [
+    "Connect",
+    "Collect",
+    "Optimize",
+]
+
+
+class StrictSchemaValidator(BaseValidator[ContentTypes]):
+    error_code = "ST111"
+    description = "Validate that the scheme's structure is valid, no fields excluded."
+    rationale = "Maintain valid structure for content items."
+    expected_git_statuses = [GitStatuses.MODIFIED, GitStatuses.ADDED]
```

Comment:
why do we skip validating renamed files?
I think we can just remove this field and set the validation to run with -g